sys/net/gnrc/network_layer/ipv6/nib/_nib-6lr.c

```diff
@@ -69,6 +69,21 @@ uint8_t _reg_addr_upstream(gnrc_netif_t *netif, const ipv6_hdr_t *ipv6,
                 _handle_sl2ao(netif, ipv6, icmpv6, sl2ao);
                 /* re-get NCE in case it was updated */
                 nce = _nib_onl_get(&ipv6->src, netif->pid);
+                /* and re-check EUI-64 in case nce was not an NC before */
```

Comment:
What's an NC?

Reply:
Neighbor cache. This refers to the NC flag in the NIB. If you read the code just a few lines above, you notice it is checked ;-).

Reply:
https://github.com/RIOT-OS/RIOT/pull/14883/files#diff-53803937a78b3e708c1550cac8cdc75dR63